ChopChopTests/RecipeParserTest.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  RecipeParserTest.swift
+//  ChopChopTests
+//
+//  Created by Cao Wenjie on 14/3/21.
+//
+
+import XCTest
+@testable import ChopChop
+
+class RecipeParserTest: XCTestCase {
+
+    func testParseIngredientList() throws {
+        let ingredients = [
+            "2 tablespoon olive oil",
+            "1 lemon, juiced"
+        ]
+        let result = RecipeParser.parseIngredientList(ingredientList: ingredients)
+        let right: [String: Quantity] = [
+            "olive oil": .volume(0.015 * 2),
+            "lemon, juiced": .count(1)
```

Comment:
How should we deal with a situation like "lemon, juiced"? I think this is something the team needs to discuss